### PR TITLE
add ignore rules for empty-line-before

### PR DESCRIPTION
**Pull request by mcallegari10**
## Description

Add ignore rules for spacing issues. It's becoming really annoying the use of this rules when trying to import and use SASS functions

### CODE REVIEW

**Review by FrankIglesias — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = {
     "scss/no-duplicate-dollar-variables": true,
     "at-rule-blacklist": null,
     "at-rule-empty-line-before": ["always", {
-      "except": ["after-same-name"]
+      "ignoreAtRules": ["function", "if", "each", "include", "mixin", "for", "else", "import", "media"]
```
Comment (damfinkel) [inline index.js]: I can't find in the docs most of this options for `ignoreAtRules`: https://stylelint.io/user-guide/rules/at-rule-empty-line-before

File: index.js [hunk 2]
```diff
@@ -156,7 +156,8 @@ module.exports = {
     "property-no-vendor-prefix": true,
     "property-whitelist": null,
     "rule-empty-line-before": ["always", {
-      "except": ["first-nested"]
+      "except": ["first-nested"],
+      "ignoreAtRules": ["function", "if", "each", "include", "mixin", "for", "else", "import", "media"]
```
Comment (damfinkel) [inline index.js]: I can't find `ignoreAtRules` in this rule's docs, I think it's `ignore`: https://stylelint.io/user-guide/rules/rule-empty-line-before

File: index.js (hunk 1 above)
Comment (mcallegari10) [reply index.js]: https://stylelint.io/user-guide/rules/at-rule-empty-line-before#ignoreatrules-array-of-at-rules

File: index.js (hunk 2 above)
Comment (mcallegari10) [reply index.js]: Yup

**Review by damfinkel — APPROVED** [review]

**Pull request merged**